scss/style.scss

```diff
@@ -0,0 +1,130 @@
+@import "reset";
+@import "color";
+@import "mixin";
+
+.main {
+  @include size(100%, 100vh);
+  @include flex(unset, center, center);
+  background-color: $lightCyan;
+}
+.grid {
+  display: grid;
+  grid-template-columns: repeat(3, 1fr);
+  column-gap: 5%;
+  @include size(65%, 80%);
+  &__item {
+    background-color: $white;
+    position: relative;
+    height: 100%;
+    &::before {
+      content: "NEW";
+      @include size(60px, 60px);
+      @include flex(row, center, center);
+      background-image: url("../assets/images/newmark.png");
+      background-size: cover;
+      position: absolute;
+      z-index: 5;
+      transform: translate(-20%, -30%);
+      @include textFont($white, 12px, none);
+      display: none;
+    }
+    &:nth-child(2)::before {
+      display: flex;
+    }
+  }
+  &__item-price {
+    @include size(100px, 100px);
+    @include itemPrice($white);
+    @include flex(column, center, center);
+    position: absolute;
+    z-index: 2;
+    left: 50%;
+    transform: translate(-50%, -50%);
+    border-radius: 50%;
+    font-weight: 300;
+  }
+  &__item-price--mountain {
+    background-color: $black;
+  }
+  &__price-value {
+    font-size: 22px;
+    margin-bottom: 6px;
+  }
+  &__price-value--ocean {
+    color: $blue;
+  }
+  &__price-value--mountain {
+    color: $darkPink;
+  }
+  &__price-value--forest {
+    color: $green;
+  }
+  &__price-desc {
+    font-size: 12px;
+  }
+  &__item-info {
+    text-align: center;
+    height: 100%;
+  }
+  &__item-place {
+    @include size(100%, 38%);
+    @include flex(column, center, center);
+    filter: brightness(0.9);
+  }
+  &__item-place--ocean {
+    @include placeBgImage("../assets/images/ocean.png", $opacityColor);
+  }
+  &__item-place--mountain {
+    @include placeBgImage("../assets/images/mountain.png", $opacityColor);
+  }
+  &__item-place--forest {
+    @include placeBgImage("../assets/images/forest.png", unset);
+  }
+  &__place-desc {
+    @include textFont($white, 30px, 200);
+    font-family: "Poppins";
+    letter-spacing: 4px;
+  }
+  &__item-note {
+    @include textFont($lightBlue, 12px, 300);
+  }
+  &__item-desc {
+    @include textFont($mediumBlue, unset, 300);
+    height: 45%;
+  }
+  &__item-title-container {
+    height: 24%;
+    @include flex(column, center, unset);
+  }
+  &__item-title {
+    font-size: 18px;
+    padding-bottom: 5px;
+  }
+  &__item-btn {
+    height: 17%;
+    @include flex(unset, center, center);
+  }
+  &__item-ocean-btn {
+    @include button($blue);
+  }
+  &__item-mountain-btn {
+    @include button($purple);
+  }
+  &__item-forest-btn {
+    @include button($green);
+  }
+}
+.place-inf {
+  list-style-type: none;
+  font-size: 13px;
+  height: 76%;
+  @include flex(column, space-between, unset);
+  :nth-child(odd) {
+    background-color: $veryLightBlue;
+  }
+  &__detail {
+    flex: 1;
+    @include flex(unset, center, center);
+  }
+}
+@import "style_responsive";
```

Comment:
E check lại responsive nhá. A thấy chưa ổn đâu. Trên màn hình tablet đang to quá, còn xuống mobile thì đang bị lỗi dưới 374px.

##Tablet:
![Screenshot 2024-03-27 at 11 37 19](https://github.com/Dang3032k2/price_table/assets/54623466/085b4cb1-1ead-4cab-ace7-43c6e48eaf95)

##Mobile
![Screenshot 2024-03-27 at 11 37 03](https://github.com/Dang3032k2/price_table/assets/54623466/c44bb300-701b-428c-aa8c-d43dfbc5de15)